render-tests/cjk-fonts/literal/style.json

```diff
@@ -0,0 +1,44 @@
+{
+  "version": 8,
+  "metadata": {
+    "test": {
+      "height": 512,
+      "width": 512
+    }
+  },
+  "center": [
+    -90.63720703125,
+    39.28329180414049
+  ],
+  "zoom": 13,
+  "sources": {
+    "geojson": {
+      "type": "geojson",
+      "data": "local://data/bbox.geojson"
+    }
+  },
+  "sprite": "local://sprites/sprite",
+  "glyphs": "local://glyphs/{fontstack}/{range}.pbf",
+  "layers": [
+    {
+      "id": "background",
+      "type": "background",
+      "paint": {
+        "background-color": "white"
+      }
+    },
+    {
+      "id": "icon",
+      "type": "symbol",
+      "source": "geojson",
+      "layout": {
+        "icon-image": "dog-park-12",
+        "text-field": "{name}",
+        "text-font": [
+          "NotoCJK"
+        ],
+        "symbol-placement": "point"
+      }
+    }
+  ]
+}
```

Comment:
Is the expected output correct? Based on the geojson fixtures I would expect to see CJK glyphs not, for example, roman numerals.